redisinsight/api/src/modules/database/repositories/local.database.repository.ts

```diff
@@ -90,6 +92,17 @@ export class LocalDatabaseRepository extends DatabaseRepository {
    * @param database
    */
   public async create(database: Database): Promise<Database> {
+    // Do not create a connection if it triggered from cloud and have the same fields
+    if (database.cloudDetails) {
```

Comment:
is it better to check "cloud database id" in the cloudDetails and only then other fields (may be even without host and port)